benchmarking-cli/src/utils_program.rs

```diff
@@ -0,0 +1,48 @@
+use miden_stdlib::StdLibrary;
+use miden_vm::{Assembler, Kernel, Program, ProgramInfo};
+
+pub struct MidenProgram {
+    pub assembler: Assembler,
+    pub masm_code: String,
+    pub debug: bool,
+    pub program: Option<Program>,
+    pub program_info: Option<ProgramInfo>,
+}
+
+pub const DEBUG_OFF: bool = false;
+//pub const DEBUG_ON: bool = true;
```

Comment:
Suggestion: This allows the caller site to be more readable, `MidenProgram::new(prog, DEBUG_OFF);` instead of `MidenProgram::new(prog, false);`. But that is only the case if the user knows about the constant. Instead if you use:

```rust
enum Debug {
  On,
  Off,
}
```

We always get the nicer readability.

Reply:
I removed it now, I think, no one wants to benchmark in DebugMode